base/doc/ltnews41.tex

```diff
@@ -452,6 +452,25 @@ \subsection{\pkg{array}:\ Improve preamble setup code for \texttt{p} and friends
 \githubissue{1585}
 
 
+\subsection[\pkg{varioref}:\ How to make \cs{reftex...}\ empty]
+           {\pkg{varioref}:\ How to make \cs{reftexafter} and friends empty}
+
+In the unusual case that one wants to make a command such as
+\cs{reftextafter} produce nothing one has o get rid of the space that
+is automatically placed in front of it. This can be done by simply
```

Comment:
I think this means:
it.  ==>   the command.

Reply:
is that really unclear?

Reply:
"it" is often unclear: why not avoid any misunderstanding?